test/sanity/secrets.yaml

```diff
@@ -1,3 +1,6 @@
 NodeStageVolumeSecret:
     username: sanity
     password: sanitytestpassword
+NodePublishVolumeSecret:
```

Comment:
this is not necessary since for smb csi driver, it only requires secret in `NodeStageVolume`

Reply:
But in sanity test, it has no secret in createVolume req, so it return error as    
```  
rpc error: code = Internal desc = failed to mount src nfs server: rpc error: code = Internal desc = volume(127.0.0.1/share#sanity-controller-source-vol-6244697B-71F35652#) mount "//127.0.0.1/share" on "/tmp/sanity-controller-source-vol-6244697B-71F35652" failed with mount failed: exit status 1
      Mounting command: systemd-run
      Mounting arguments: --description=Kubernetes transient mount for /tmp/sanity-controller-source-vol-6244697B-71F35652 --scope -- mount -t cifs -o <masked> //127.0.0.1/share /tmp/sanity-controller-source-vol-6244697B-71F35652
      Output: Running scope as unit: run-r21304af4cc59421f926ae82a5143fd93.scope
      username specified with no parameter
``` 
in `internalMount` func
https://github.com/kubernetes-csi/csi-driver-smb/actions/runs/4783827931/jobs/8504552148?pr=601

Reply:
in Line 2122

Reply:
so I use CreateVolumeSecret